web/satellite/src/components/account/referral/ReferralArea.vue

```diff
@@ -0,0 +1,397 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+	<div class="referral-container">
+		<NoProjectState v-if="isNoProjects"/>
+		<div class="referral-container__wrapper" v-if="!isNoProjects">
+			<p class="referral-container__title">Refer A Friend And You'll Both Get Free Credits</p>
+			<p class="referral-container__subtitle">You Get X GB, They Get X GB</p>
+			<div class="referral-container__icons">
+                <div class="icon-container">
+				    <img src="../../../../static/images/referral/send.svg">
+                    <p class="referral-container__icons__description">Copy and share your invite link</p>
+                </div>
+                <div class="icon-container">
+                    <img src="../../../../static/images/referral/coins.svg">
+				    <p class="referral-container__icons__description">They get XGB in free credits</p>
+                </div>
+                <div class="icon-container">
+                    <img src="../../../../static/images/referral/calendar.svg">
+				    <p class="referral-container__icons__description">They use our network for 60 days</p>  
+                </div>
+                <div class="icon-container">
+                    <img src="../../../../static/images/referral/money.svg">
+				    <p class="referral-container__icons__description">You get XGb in free credits</p>
+                </div>
+			</div>
+			<div class="referral-container__copy-and-share-container">
+				<p class="referral-container__copy-and-share-container__title">Copy And Share Your Invitation Link</p>
+				<div class="referral-container__copy-and-share-container__link-holder">
+					<p class="referral-container__copy-and-share-container__link-holder__link">test://test/test/test/test</p>
```

Comment:
FYI, this is the logic for referral link: https://github.com/storj/storj/blob/bdcb40fbc8b64eb315980cfe17ace5af804df6b4/satellite/rewards/partners.go#L78.